crates/bevy_ecs/src/storage/table.rs

```diff
@@ -90,26 +101,22 @@ impl Column {
         row: usize,
     ) -> (*mut u8, ComponentTicks) {
         let data = self.data.swap_remove_and_forget_unchecked(row);
-        let ticks = (*self.ticks.get()).swap_remove(row);
+        let ticks = self.ticks_mut().swap_remove(row);
         (data, ticks)
     }
 
-    /// # Safety
-    /// allocated value must be immediately set at the returned row
-    pub(crate) unsafe fn push_uninit(&mut self) -> usize {
+    // Safety:
```

Comment:
```suggestion
    # Safety
```

Generally the convention across the code base is `# Safety` format. 
(I'm pretty sure this is even what `clippy` recommends)